src/components/AppLayout/Content.tsx

```diff
@@ -6,12 +6,25 @@ import { borderHeight, savebarHeight, topBarHeight } from "./consts";
 interface ContentProps {
   [key: `data-${string}`]: string;
   children: React.ReactNode;
+  noSavebar?: boolean;
+  noTopNav?: boolean;
 }
 
-export const Content: React.FC<ContentProps> = ({ children, ...rest }) => (
+export const Content: React.FC<ContentProps> = ({
+  children,
+  noSavebar = false,
+  noTopNav = false,
+  ...rest
+}) => (
   <Box
     __gridArea="content"
-    __height={`calc(100vh - ${topBarHeight} - ${savebarHeight} - ${borderHeight})`}
+    __height={
+      noSavebar
+        ? `calc(100vh - ${topBarHeight} - ${borderHeight})`
+        : `calc(100vh - ${
+            noTopNav ? "0px" : topBarHeight
+          } - ${savebarHeight} - ${borderHeight})`
+    }
```

Comment:
suggestion: it looks bit complicated, maybe will be better to move this to function

Reply:
I agree - but what do you think about refactoring it in the next PR? I want to fix layout asap and then do proper refactor.

Reply:
+1 

```js

const createHeight = ({ topBarHeight, borderHeight })  => {

 const withoutSaveBar = () => {
   return `calc(100vh - ${topBarHeight} - ${borderHeight})`
 }

 const withSaveBar = ({ noTopNav }) => {
  const topHeight = !noTopNav &&  `- ${topBarHeight}`

   return `calc(100vh - ${topHeight} - ${savebarHeight} - ${borderHeight})`
 }

  return { withoutSaveBar, withSaveBar }
}


const { withoutSaveBar, withSaveBar } = createHeight({ topBarHeight, borderHeight })
...
__height={noSavebar ? withoutSaveBar() :  withSaveBar({ noTopNav })}

```